packages/main/src/themes/ListItemBase.css

```diff
@@ -9,6 +9,33 @@
 	border-bottom: 1px solid transparent;
 }
 
+/* highlight */
+.ui5-li-highlight {
+	position: absolute;
+    width: 0.375rem;
+    bottom: 0;
+    left: 0;
+    top: 0;
+    border-right: 0.0625rem solid #fff;
```

Comment:
- border-right won't work in RTL
- isn't there a param for #fff (if not, we could ask designers to suggest one, in the name of the progress we can merge it like this if white works fine for all themes)